index.html

```diff
@@ -10244,6 +10262,43 @@ <h3>State change notification</h3>
 		<h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<p>Below is an alphabetical list of <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> <a>states</a> and [=ARIA/properties=] to be used by rich internet application authors. A detailed definition of each <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> state and [=ARIA/property=] follows this compact list.</p>
 		<p id="index_state_prop" class="placeholder">Placeholder for index of states and properties</p>
+		<div class="property" id="aria-actions">
+			<pdef>aria-actions</pdef>
+			<div class="property-description">
+				<p><a>Identifies</a> a related element or elements whose primary activation (<code>click</code> event) will trigger a behavior or operation relevant to the referencing user interface object, such as the close button for a dialog, or a reply button relevant to the email message that references it.</p>
+				<p>The <code>aria-actions</code> attribute allows an element to reference other interactive elements which trigger actions on the current element. For example, in a web mail application, the currently selected email message could reference action elements such as the <em>message-specific</em> Reply, Forward, and Delete buttons. Actions are triggered by mainstream pointer events (click, tap) or by simulated pointer events from assistive technology.</p>
+				<ul>
+					<li>Authors MUST ensure that elements referenced by <code>aria-actions</code> have an accessible name.</li>
+					<li>Authors MUST ensure that related action elements respond to a <code>click</code> event and are not limited to modality-specific activation such as the <code>keyup</code> or <code>touchend</code> events.</li>
+					<li>Authors MUST ensure the secondary action is either directly navigable with the keyboard, or that there is a keyboard shortcut to directly activate it when focus is on the referencing element.</li>
```

Comment:
This is the first appearance of the term "secondary action". I don't think a normative statement should use a term that isn't defined.

```suggestion
					<li>Authors MUST ensure each referenced action element is either directly navigable with the keyboard or that there is a keyboard shortcut to directly activate it when focus is on the referencing element.</li>
```